<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>60</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>204</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="whatsThis">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">font: 900 italic 22pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Men√∫ mesero</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>204</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QFrame" name="frame_4">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>89</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QFrame" name="frame_5">
                 <property name="minimumSize">
                  <size>
                   <width>400</width>
                   <height>300</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>400</width>
                   <height>400</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true"/>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <widget class="QPushButton" name="bt_mesa1">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>30</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="styleSheet">
                   <string notr="true"/>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="bt_mesa2">
                  <property name="geometry">
                   <rect>
                    <x>160</x>
                    <y>30</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="bt_mesa3">
                  <property name="geometry">
                   <rect>
                    <x>300</x>
                    <y>30</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="bt_mesa5">
                  <property name="geometry">
                   <rect>
                    <x>160</x>
                    <y>180</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="bt_mesa4">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>180</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="bt_mesa6">
                  <property name="geometry">
                   <rect>
                    <x>300</x>
                    <y>180</y>
                    <width>85</width>
                    <height>80</height>
                   </rect>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>85</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>0</width>
                    <height>80</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>iconos interfaz/mesa.png</normaloff>iconos interfaz/mesa.png</iconset>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>80</width>
                    <height>80</height>
                   </size>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m1">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>120</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 1</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m2">
                  <property name="geometry">
                   <rect>
                    <x>160</x>
                    <y>120</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 2</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m3">
                  <property name="geometry">
                   <rect>
                    <x>300</x>
                    <y>120</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 3</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m4">
                  <property name="geometry">
                   <rect>
                    <x>20</x>
                    <y>270</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 4</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m5">
                  <property name="geometry">
                   <rect>
                    <x>160</x>
                    <y>270</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 5</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                 <widget class="QLabel" name="lb_indicador_m6">
                  <property name="geometry">
                   <rect>
                    <x>300</x>
                    <y>270</y>
                    <width>81</width>
                    <height>20</height>
                   </rect>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">font: 900 italic 10pt &quot;Segoe UI&quot;;</string>
                  </property>
                  <property name="text">
                   <string>MESA 6</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>89</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_salir">
            <property name="styleSheet">
             <string notr="true">font: 900 italic 18pt &quot;Segoe UI&quot;;</string>
            </property>
            <property name="text">
             <string>Salir</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
